src/Geta.404Handler/Core/CustomRedirects/CustomRedirect.cs

```diff
@@ -35,6 +35,9 @@ public string OldUrl
 
         public int  State { get; set; }
 
+        // 301 (permanent) or 302 (temporary)
+        public int RedirectType { get; set; }
```

Comment:
Instead of codes, it would be better to use `enum` with values - `Permanent` and `Temporal` with `int` values - 301 and 302. Then in the database, it can be stored as `int`. Everywhere else - would use `enum` values.

Reply:
I will update this to do like with the State property above (enum for Redirect Type)

Reply:
This property should also be of `RedirectType` enum. You should convert from/to enum in the `SqlRedirectRepository`.